sbibm/algorithms/abcpy/rejection_abc.py

```diff
@@ -0,0 +1,166 @@
+from typing import Optional, Tuple
+
+import numpy as np
+import torch
+from abcpy.backends import BackendDummy
+from abcpy.inferences import RejectionABC
+from abcpy.statistics import Identity
+
+import sbibm
+from sbibm.algorithms.abcpy.abcpy_utils import (
+    ABCpyPrior,
+    ABCpySimulator,
+    get_distance,
+    journal_cleanup_rejABC,
+)
+from sbibm.tasks.task import Task
+from sbibm.utils.io import save_tensor_to_csv
+from sbibm.utils.kde import get_kde
+from sbibm.utils.torch import sample
+
+
+def run(
+    task: Task,
+    num_samples: int,
+    num_simulations: int,
+    num_simulations_per_param: Optional[int] = 1,
+    num_observation: Optional[int] = None,
+    observation: Optional[torch.Tensor] = None,
+    num_top_samples: Optional[int] = 100,
+    quantile: Optional[float] = None,
+    eps: Optional[float] = None,
+    distance: str = "l2",
+    save_distances: bool = False,
+    sass: bool = False,
+    sass_fraction: float = 0.5,
+    sass_feature_expansion_degree: int = 3,
+    kde_bandwidth: Optional[str] = None,
+) -> Tuple[torch.Tensor, int, Optional[torch.Tensor]]:
+    """Runs REJ-ABC from `ABCpy`
+
+    We do not implement SASS for now, and LRA after sampling.
```

Comment:
this sentence is not clear to me. do you mean that LRA will not be implemented at all and SASS is not yet? Then a suggestion:
"ABCpy does not implement LRA post processing. SASS is supported in ABCpy but not yet implemented here. "

Reply:
I fixed that. Sorry, it was very unclear.